### PR TITLE
Navigation drawer resolved

### LINKED ISSUE #2166: Selected item and displayed item differ in Navigation drawer
**Actual Behaviour**
1) When we return back to the app  after we rate in playstore FAQ's are displayed but the navigation drawer has "Buy PSLab" as selected item.

Screenshot from 2021-01-13 14-32-04

https://user-images.githubusercontent.com/70392921/104429952-29442600-55ac-11eb-8eda-f955cf94b331.mp4

2) "Buy PSLab" is displayed as selected after we have closed the URL popup.

https://user-images.githubusercontent.com/70392921/104702528-21b38700-573c-11eb-9e28-9ab97104682f.mp4
 
**Expected Behaviour**
FAQ should be the selected item since faq's are displayed to the user.

### PR DESCRIPTION
**Pull request by asaikarthikeya**
Fixes #2166 

**Changes**: 
1.Set navItemIndex=8 for the case CURRENT_TAG= TAG_FAQ 

https://user-images.githubusercontent.com/70392921/104429245-6956d900-55ab-11eb-9e73-ecd7665f945e.mp4

2. Added Rate App in strings of each language.

https://user-images.githubusercontent.com/70392921/104703014-d188f480-573c-11eb-8694-83fa4306ee71.mp4

### CODE REVIEW

**Comment by asaikarthikeya** [comment]
@CloudyPadmal Please review the changes . Thank you

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 1]
```diff
@@ -187,7 +187,7 @@ private Fragment getHomeFragment() throws IOException {
                 return HomeFragment.newInstance(ScienceLabCommon.scienceLab.isConnected(), ScienceLabCommon.scienceLab.isDeviceFound());
             case 5:
                 return AboutUsFragment.newInstance();
-            case 7:
+            case 8:
```
Comment (CodeWithSouma) [inline app/src/main/java/io/pslab/activity/MainActivity.java:190]: why you change this case 7 to case 8 ?

Comment (asaikarthikeya) [reply app/src/main/java/io/pslab/activity/MainActivity.java:190]: In navigationdrawer , Rate app is in 7th position(count begins from 0) , After adding that in strings.xml new position of faq is 8.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
I noticed that when we get back to the app from Play Store, FAQ is opened. Why can't it be the same window where the user was at when clicking the `Rate` option?

**Comment by asaikarthikeya** [comment]
> I noticed that when we get back to the app from Play Store, FAQ is opened. Why can't it be the same window where the user was at when clicking the `Rate` option?

"Rate app" does not have any activity/Fragment inside the app. The content displayed on main screen should be same as the of selected item in navigation menu. Hence when FAQ section is displayed on main screen FAQ text is highlighted.

**Comment by CloudyPadmal** [comment]
This is fixed by #2199

**Comment by asaikarthikeya** [comment]
> This is fixed by #2199

I thought it was intentionally made to show FAQs after Rating app. Since FAQs have most of the questions which they may express in reviews in playstore ?? Thank you

**Comment by asaikarthikeya** [comment]
@CloudyPadmal  Sir, The mentioned PR doesnot resolve this error. 

https://user-images.githubusercontent.com/70392921/113270887-920f8480-92f7-11eb-9041-e22454840a9a.mp4

**Pull request closed without merging**